velero-plugins/pod/restore.go

```diff
@@ -32,39 +32,44 @@ func (p *RestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (*v
 	itemMarshal, _ := json.Marshal(input.Item)
 	json.Unmarshal(itemMarshal, &pod)
 	p.Log.Infof("[pod-restore] pod: %s", pod.Name)
+	
+	if (input.Restore.Labels[common.MigrationApplicationLabelKey] == common.MigrationApplicationLabelValue) && (input.Restore.Annotations[common.MigrateCopyPhaseAnnotation] == "stage"){
```

Comment:
Another minor comment. This line is too long -- it's probably best to split after the `&&`
@dymurray I forget the exact max length we're using for coding conventions -- is it 120?

Reply:
Does this look good?

if (input.Restore.Labels[common.MigrationApplicationLabelKey] == common.MigrationApplicationLabelValue) {
		if (input.Restore.Annotations[common.MigrateCopyPhaseAnnotation] == "stage") {
			pod.Labels[common.MigratePodStageLabel] = "true"
			pod.Spec.Affinity = nil
		}
	} else {

Reply:
I was thinking more like this:
```
	if (input.Restore.Labels[common.MigrationApplicationLabelKey] == common.MigrationApplicationLabelValue) &&
		(input.Restore.Annotations[common.MigrateCopyPhaseAnnotation] == "stage") {
```
We still want the same else clause to run regardless of which of the two comparisons fails. Same logic as what's in the PR, but with a line break.

Reply:
oh okay!